packages/components/src/FormField/FormField.tsx

```diff
@@ -166,13 +186,23 @@ export const FormField = React.forwardRef(
       handleValidation();
     }, [value]);
 
+    let hasErrors = false;
```

Comment:
Was just thinking we could make a little function that would return `hasErrors` this would let us do something like `const hasErrors = areThereMessages(validations)` here which would simplify reading this component and then isolate this logic into a method. Also, it's an excuse to use VSCode's fancy refactor tools!